Release 9.4.0

clairesunstudio opened the pull request:
## 9.4.0 (03/28/2019)

### Added
- (Docs) Add an issue template for submitting Mayflower project to the implementation listing #520

### Changed
- (React) [*] DP-12932: Upgrades storybook and related add-ons to version 5.0.x. #516
- (React) [*] DP-12932: Adds storysource and a11y add-ons. #516

legostud commented on styleguide/source/_patterns/02-molecules/download-link.twig:
```diff
@@ -14,6 +14,7 @@
     {% else %}
       <a 
         class="ma__download-link__file-link" 
+        aria-label="Open {{ downloadLink.format }} file{% if downloadLink.size %}, {{ downloadLink.size }},{% endif %} for {{ downloadLink.decorativeLink.text }}"
```
Instead of using aria-label on an `a` tag, it would be better to just use a `title` attribute.  Titles will provide information to everyone not just screen readers.  Note: title only works on `a` tags for screen readers.

iansholtys commented:
In this instance I'm not sure there's a need to expose the information to other devices using `title` as it should all be readily available to sighted users. Also, when using OS X's VoiceOver utility, the title is read after the link's normal text, which isn't very helpful.

Because we're just prefacing the normal link text (the name of the file) with some additional information (the action, and file type/size), the simpler approach here is to just use the `visually-hidden` class on a span inside the link. This should behave consistently across all screen readers.

clairesunstudio committed:
Merge pull request #514 from massgov/master

Master

clairesunstudio committed:
GitBook: [develop] 25 pages and 107 assets modified

clairesunstudio committed:
Merge pull request #515 from massgov/master

Sync Master into Develop

smurrayatwork committed:
React/DP-12932:  Storybook v5 upgrade. (#516)

* Upgrades storybook to version 5.0.x, adds accessibility, storysource addons.

* Updates backstop, fixes backstop configuration for storybook 5 paths.

* Updates to 5.0.5

* Adds changelog.

* update button colour for add-info

* Fixes decorator for withInfo to be global.

* remove with info decorator from individual stories

* bug fix

* bug fix

* move storybook with info bottom to bottom right

* update reference images

* Update changelogs/DP-12932.md

* Update changelogs/DP-12932.md

* testing delay

* test again

* update circle node version

* update package json

* test webpack cahnge temporarily

* Revert "test webpack cahnge temporarily"

This reverts commit 7710db4973dbe1474e7368d18e78c5bccd86ab8b.

* test reverting webpack config

* Revert "test reverting webpack config"

This reverts commit 89734d893dcc77a452c0855660fd727c950431e2.

* Reverts backstopjs version.

* Update react/backstop/backstop.js

clairesunstudio committed:
New issue template

clairesunstudio committed:
Add an issue template (#520)

* Add an issue template

Add an issue template for submitting Mayflower project to the implementation listing

* update url

clairesunstudio committed:
add changelog

clairesunstudio committed:
consolidate changelog
